cms/djangoapps/contentstore/views/course.py

```diff
@@ -469,6 +474,52 @@ def _get_rerun_link_for_item(course_key):
     return reverse_course_url('course_rerun_handler', course_key)
 
 
+def _deprecation_info(course_module, deprecated_block_types):
+    """
+    Returns information about ORA1
+        * Modules presence in the Advanced Module List.
+        * Components presence in the course outline.
+
+    Arguments:
+        course_id (CourseKey): course id
+        advanced_modules (list): advance module list
+
+    Returns:
+        Dict with following keys:
+
+        ora1_enabled (bool): True if "peergrading" and/or "combinedopenended" are in the Advanced Module List else False
+        ora1_components (list): List of ora1 components and their parent's url
+        advance_settings_url (str): URL to advance settings page
+    """
+    data = {
+        'feature_enabled': any(
+            component in course_module.advanced_modules for component in deprecated_block_types
+        ),
+        'feature_components': [],
+        'advance_settings_url': reverse_course_url('advanced_settings_handler', course_module.id)
+    }
+
+    try:
+        structure_data = api.course_structure(course_module.id, block_types=deprecated_block_types)
+    except errors.CourseStructureNotAvailableError:
+        return data
+
+    store = modulestore()
+    components = []
+    for __, block in structure_data['blocks'].items():
```

Comment:
If we're ignoring the keys anyway, why not use `.values()` instead of `.items()`?

Also, does this give you the ordering you want?